contracts/governance/script/DeployToken.s.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import "forge-std/console2.sol";
+
+contract Deploy is Script {
+    address internal constant DEPLOYER = 0xd56716E3cC851bAF569795351917df73F8762A7F;
+
+    function setUp() public {}
+
+    function run() external {
+        vm.startBroadcast(vm.envUint("PRIVATE_KEY"));
```

Comment:
You have the private key used to deploy ZRX saved somewhere? Otherwise, I guess we could use a combination of `vm.record()`, `vm.store()`, and `vm.etch()` to get the contract "deployed" to the right address?

Reply:
It seems to be the [standard ZrxDeployer account](https://etherscan.io/address/0xa3b2d1087bcebe59d188a23f75620612d967df72) who rolled the token to production.

Reply:
Leaving a note here that's burnt.